folium/utilities.py

```diff
@@ -356,3 +351,38 @@ def camelize(key):
     """
     return ''.join(x.capitalize() if i > 0 else x
                    for i, x in enumerate(key.split('_')))
+
+
```

Comment:
These utility functions would be easy to write tests for.

Reply:
Indeed we need to test the functions in `utilities.py`. I would leave that to another PR though. This one drifted from its purpose a lot already :grimacing:

Reply:
That's okay :)